spec/models/candidate_interface/personal_details_form_spec.rb

```diff
@@ -1,6 +1,34 @@
 require 'rails_helper'
 
 RSpec.describe CandidateInterface::PersonalDetailsForm, type: :model do
+  let(:data) do
+    {
+      first_name: Faker::Name.first_name,
+      last_name: Faker::Name.last_name,
+      date_of_birth: Faker::Date.birthday,
+      first_nationality: NATIONALITIES.sample,
```

Comment:
it is fine to use this, I just wonder why we do not use the faker as well.

Reply:
The faker blows up because `Faker::Nation` is not the same as our own `NATIONALITIES` 👍